core/vocabulary/vocabulary.py

```diff
@@ -239,7 +230,8 @@ def append_to_vocabulary(rows: dict[str, str]):
             if key == value:
                 file.write(f"{key}\n")
             else:
-                value = repr(value)
+                if not str.isprintable(value) or "'" in value or '"' in value:
```

Comment:
I'm using the same logic used elsewhere:
https://github.com/talonhub/community/blob/13ba0ede79867ced6d309eb32fff5a00db205af3/migration_helpers/migration_helpers.py#L160